Tutorials/Level1Tutorial.md

```diff
@@ -0,0 +1,793 @@
+a single SST component.  The first step in creating a new simulation input 
+is to utilize the `import sst` statement.  This ensures that all the required 
+Python functions for SST are enabled.  Users may also utilize additional Python 
+import statements in order to enable additional Python-derived functionality 
+within the simulation script.  In this manner, simulation scripts are **not** 
+limited to SST functionality.  All normal Python syntatic operations may 
+be utilized as with a traditional Python script.  
+
+The next section includes the SST Core Options.  These options drive the 
+top-level simulation configuration that are utilized across components.  Each 
+of the core program options are set with the Python function 
+`sst.setProgramOptions("Option", "Value)`.  The two most commonly 
+used program options are the `timebase` and `stop-at` options.  The 
+`timebase` option sets the baseline global clock cadence by which 
+SST will govern the core simulation components.  Each component has the ability 
+to set higher or lower fidelity clocks governing local event timing, but the 
+global clock drives the top-level simulation cadence.  Further, the `stop-at` 
+program option determines when the simulation will complete reletive 
+to the core timebase clock frequency.  In our example below we set a 
+`timebase` of 1 picosecond and inform SST to stop the simulation at 10000 
+seconds of simulated time.  Users may also specify a value of `0` 
+for the `stop-at` program option.  This tells the SST core to wait until 
+all components and sub-components have signaled completion to the core 
+before stopping the simulation.  Note that for these options, 
+SST automatically interprets the labeled units.  In this manner, users 
+may utilize traditional labels such as `ns`, `us`, `ms` and `s` to specify 
+timing values in nanoseconds, microseconds, milliseconds and seconds, 
+respectively.  This is also true for specifying traditional measures 
+of capacity (`GB`,`KB`, etc).  
+
+The full list of permissible global program options are listed as follows:
+
+|  **Option**  | **Description** | **Values** |
+|:-|:-|:-|
+| `timebase`          | The baseline clock cadence | Time (`ps`,`ns`,etc) |
+| `stop-at`           | Simulation termination point | Simulated time (`ms`, `s`, etc) |
+| `debug-file`        | Specifies the file for debug outptu | Filename |
+| `partitioner`       | Select the graph partitioner to use | `lib.partitionerName` |
+| `verbose`           | Print information about the core runtime | Boolean (`True`, `False`)|
+| `dump_partition`    | Output the SST paritioned graph info | Boolean (`True`, `False`) |
+| `dump_config_graph` | Output the SST config graph | Boolean (`True`, `False`) |
+| `model_options`     | Provide options to the Python script | String |
+| `run-mode`          | Set the run mode | `init`, `run` or `both` |
+
+The final required section in the SST simulation script involves the definition 
```

Comment:
The program options are not required, but this implies there are three required sections, not two (import sst and component config). Can you reword?

Reply:
reworded and noted that program options are optional; now two required sections